src/components/AppLayout/TopNav/TopNav.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Text } from "@saleor/macaw-ui/next";
+import React, { PropsWithChildren } from "react";
+
+import useAppChannel from "../AppChannelContext";
+import AppChannelSelect from "../AppChannelSelect";
+import { TopNavLink } from "./TopNavLink";
+import { TopNavWrapper } from "./TopNavWrapper";
+
+interface TopNavProps {
+  title: string | React.ReactNode;
+  href?: string;
+}
+
+export const TopNav: React.FC<PropsWithChildren<TopNavProps>> = ({
```

Comment:
shouldnt component be in `apps` or `new-apps` folder?

Reply:
I don't think so - this is `TopNav` for the whole dashboard - all views